back-end/persistence/models/accounts.model.js

```diff
@@ -0,0 +1,26 @@
+const mongoose = require('mongoose');
+
+const Schema = mongoose.Schema;
+
+const accountSchema = new Schema ({
+    externalLog: {
+        type: Boolean,
+        required: true
+    },
+    email: {
+        type: String,
+        required: true
+    },
+    username: {
+        type: String,
+        required: true
+    },
+    password: {
```

Comment:
I know this is just for testing but i'm just putting it here in case I forget to mention it later.

in the future, maybe we can remove the saving of password since auth0 does this for us. Benefit is we wouldn't have to worry about masking passwords.

so i think all we need to store is the 
- username
- user id given by auth0

i'm thinking that we might not even need the email too.

Reply:
good point!! Will keep in mind when I'm doing the account stuff